Bottom: 40408ad1a4615bb2cae6ed57c68e81c1ce88f543
Top:    224df6b6c2eec5e67003c4581156a240b20b131e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-10 03:38:20 +0000

Refresh of play-with-display-lists

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 8c479f5..28a1975 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1753,8 +1753,10 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
                              GdkEventExpose *ev,
                              GHidPort *port)
 {
+#if 0
   static int one_shot = 1;
   static int display_list;
+#endif
   BoxType region;
   int eleft, eright, etop, ebottom;
   int min_x, min_y;
